### PR TITLE
Shortcode removal utilities

### PR DESCRIPTION
**Pull request by thunderer**

### CODE REVIEW

File: src/Coduo/PHPHumanizer/String/ShortcodeProcessor.php [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+use Thunder\Shortcode\HandlerContainer\HandlerContainer;
+use Thunder\Shortcode\Parser\RegularParser;
+use Thunder\Shortcode\Processor\Processor;
+use Thunder\Shortcode\Shortcode\ShortcodeInterface;
+
+class ShortcodeProcessor
+{
+    public function __construct()
+    {
```
Comment (norzechowicz) [inline src/Coduo/PHPHumanizer/String/ShortcodeProcessor.php]: I think we can remove it

Comment (thunderer) [inline src/Coduo/PHPHumanizer/String/ShortcodeProcessor.php]: @norzechowicz I think that explicit constructor is better, even if it's empty. Do you want me to remove it or can you live with it? Do you have any other issues with this PR?

Comment (norzechowicz) [inline src/Coduo/PHPHumanizer/String/ShortcodeProcessor.php]: Remove it please to make this class consistent with others we have here

**Comment by thunderer** [comment]
@norzechowicz Empty constructor was removed as requested.

**Comment by norzechowicz** [comment]
@thunderer awesome :beers: !

**Pull request merged**